### ISSUE TITLE
Phase01-Team4

### ISSUE THREAD

**Issue by Danyal0096** [Star-Academy/codestar-intern-issues#22]
This issue template is based of [Phase01-Search.md](https://github.com/Star-Academy/codestar-internship/blob/master/Projects/Phase01-Search.md) contents.
Click on "Raw" button on right side of this page to see original markdown then create a new issue per team (two interns) in [this repo](https://github.com/Star-Academy/codestar-intern-issues/issues) with orginal content of tasklist below. Your issue title should be like this: `Phase01-Team5`

- [x] Section 1: Think individually about question i & ii before starting
- [x] Section 2: Read individually about Inverted-Index
- [x] Section 3: Implement a simple inverted index with java (do pair programming)
  - [x] Implement a fileReader class which reads all files in a folder, each file is a document in your inverted-index.
  - [x] Implement an inverted-index class which contains all tokens and list of related documents per each token
  - [x] Test your program with sample dataset mentioned in project definition
 
- [x] Section4: Implement basic and, or, exclude features:
  - [x] Support for words starting with + (or feature)
  - [x] Support for words starting with nothing (and feature)
  - [x] Support for words starting with - (exclude feature)
